fendermint/vm/topdown/examples/run_polling.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+use async_stm::atomically_or_err;
+use clap::Parser;
+use fendermint_vm_topdown::{
+    Config, Error, IPCAgentProxy, IPCParentFinality, InMemoryFinalityProvider,
+    ParentFinalityProvider, ParentViewProvider, PollingParentSyncer,
+};
+use fvm_shared::address::{set_current_network, Network};
+use ipc_agent_sdk::apis::IpcAgentClient;
+use ipc_agent_sdk::jsonrpc::JsonRpcClientImpl;
+use ipc_sdk::subnet_id::SubnetID;
+use num_traits::FromPrimitive;
+use std::str::FromStr;
+use std::sync::Arc;
+use std::time::Duration;
+use tracing::Level;
+
+#[derive(Parser, Debug)]
+pub struct Options {
+    /// The URL of the ipc agent's RPC endpoint.
+    #[arg(
+        long,
+        short,
+        default_value = "http://127.0.0.1:3030/json_rpc",
+        env = "IPC_AGENT_RPC_URL"
+    )]
+    pub ipc_agent_url: String,
+
+    /// Enable DEBUG logs.
+    #[arg(long, short)]
+    pub verbose: bool,
+
+    /// The subnet id expressed a string
+    #[arg(long, short)]
+    pub subnet_id: String,
+
+    /// The subnet id expressed a string
```

Comment:
Wrong docs. If this is between mainnet and testnet, maybe it could say which value corresponds to what.

Reply:
:point_up: